src/Analyzers/MSTest.Analyzers/RoslynAnalyzerHelpers/IMethodSymbolExtensions.cs

```diff
@@ -10,6 +10,96 @@ namespace Analyzer.Utilities.Extensions;
 
 internal static class IMethodSymbolExtensions
 {
+    /// <summary>
+    /// Checks if a given symbol implements an interface member implicitly or explicitly
+    /// </summary>
+    public static bool IsImplementationOfAnyInterfaceMember(this ISymbol symbol)
+        => symbol.IsImplementationOfAnyExplicitInterfaceMember()
+        || symbol.IsImplementationOfAnyImplicitInterfaceMember();
+
+    /// <summary>
+    /// Checks if a given symbol implements an interface member explicitly
+    /// </summary>
+    public static bool IsImplementationOfAnyExplicitInterfaceMember([NotNullWhen(returnValue: true)] this ISymbol? symbol)
+    {
+        if (symbol is IMethodSymbol methodSymbol && !methodSymbol.ExplicitInterfaceImplementations.IsEmpty)
+        {
+            return true;
+        }
+
+        if (symbol is IPropertySymbol propertySymbol && !propertySymbol.ExplicitInterfaceImplementations.IsEmpty)
+        {
+            return true;
+        }
+
+        if (symbol is IEventSymbol eventSymbol && !eventSymbol.ExplicitInterfaceImplementations.IsEmpty)
+        {
+            return true;
+        }
+
+        return false;
+    }
+
+    public static bool IsImplementationOfAnyImplicitInterfaceMember(this ISymbol symbol)
```

Comment:
Not sure if this has changed since I filed https://github.com/dotnet/roslyn-analyzers/issues/5878. It may be worth actually double checking if my report is correct. But it's best if we can avoid porting bugs from roslyn-analyzers.

Reply:
I confirm this is handling both implicit and explicit.